src/components/ui/TaskFooter.jsx

```diff
@@ -0,0 +1,32 @@
+import { ReactComponent as DeleteIcon } from 'assets/delete.svg';
+import { ReactComponent as EditIcon } from 'assets/edit.svg';
+import { ReactComponent as DoneIcon } from 'assets/ok.svg';
+import Button from 'components/ui/Button';
+import propTypes from 'prop-types';
+
+export default function TaskFooter({ isCompleted = false }) {
+  return (
+    <div className="task__footer">
+      <div className="task__footer__left">
```

Comment:
```suggestion
    <div className="task_footer">
      <div className="task_footer-left">
```

Reply:
Bhai, I have renamed the class according to the BEM best practices.